### ISSUE TITLE
What's the next awesome thing we'll add to the simulator?

### ISSUE THREAD

**Issue by fmaussion** [OGGM/glacier_simulator#18]
That's a genuine question ;-)

Right now I'm for adding an option to allow for real climate parameters to be used (temp + precip) instead of linear MB, and maybe even transient random runs :star_struck:

**Comment by fmaussion** [comment]
another thing could be to select from a list of glaciers and simulate them from their pre-processed directories ;-)

**Comment by pat-schmitt** [comment]
Cool Ideas!

- I think using real climate parameters instead of a linear MB is more or less straight forward to implement. Of course, we most agree on which model to use (with its parameters, which also can be changeable with a slider) and we can maybe also include different height profiles of temperature and precipitation. And maybe create a new figure tab (beside 'geometry plots' and 'timeseries plots') showing the temperature and precipitation profiles and the resulting mass balance profile, and  in the 'geometry plot' only showing the resulting mass balance profile, in order to not make this figures more complicated.

- I do not understand what you mean with transient random runs? Random in terms of defining a mass balance timeseries and randomly picking parts of them during the model run?

- Integrating real glaciers would be the coolest thing I think, but maybe also the most labor-intensive idea. I could imagine a new menu tab 'real glaciers' (beside 'beginner mode' and 'advanced mode'). This could also include an exploration of the calculated mass balances of the pre-processed directories (with maybe one profile per year of temperature, precipitation and mass balance) in the new graphics of bullet point one of this list (maybe also including a slider to go through the individual years). But the first thing should be including the topographies before starting with the more advanced stuff.

**Comment by pat-schmitt** [comment]
A other idea could be to include the glacier run off as a variable in the timeseries plots.

**Comment by fmaussion** [comment]
Yes! Although this also requires climate data - maybe we should consider having two apps to not overblow this one... Something to consider
